src/libslic3r/PrintConfig.cpp

```diff
@@ -6120,8 +6260,16 @@ void PrintConfigDef::to_prusa(t_config_option_key& opt_key, std::string& value,
         }
     } else if ("elephant_foot_min_width" == opt_key) {
         opt_key = "elefant_foot_min_width";
-    } else if("first_layer_acceleration" == opt_key || "infill_acceleration" == opt_key || "bridge_acceleration" == opt_key || "default_acceleration" == opt_key || "perimeter_acceleration" == opt_key
-        || "overhangs_speed" == opt_key || "ironing_speed" == opt_key){
+    } else if("default_acceleration" == opt_key || "first_layer_acceleration" == opt_key 
+        || "perimeter_acceleration" == opt_key || "external_perimeter_acceleration" == opt_key 
+        || "infill_acceleration" == opt_key || "solid_infill_acceleration" == opt_key || "top_solid_infill_acceleration" == opt_key 
+        || "bridge_acceleration" == opt_key || "bridge_acceleration_internal"  == opt_key || "overhangs_acceleration" == opt_key 
+        ||  "support_material_acceleration" == opt_key ||   "support_material_interface_acceleration" == opt_key
+        || "gap_fill_acceleration" == opt_key || "thin_walls_acceleration" == opt_key
+        || "ironing_acceleration" == opt_key
+        || "travel_acceleration" == opt_key
+        || "overhangs_speed" == opt_key || "ironing_speed" == opt_key)
+    {
```

Comment:
Just noticed, that this is actually wrong. The new options shouldn't exported to Prusa of course. I will fix that.

Reply:
it's already fixed. I merge the code yesterday, just not had the tuime to push it to the dev branch before going out.